src/metabase/email/legacy_no_self_service.hbs

```diff
@@ -28,9 +28,9 @@
 
   <p>The following groups currently have their <strong>View data</strong> access set to <strong>No self-service (deprecated)</strong></p>
   <ul>
-   {{#if groups}}
-      <li><a href="{{instanceURL}}/admin/permissions/data/group/{{id}}" style="color: #336699;">{{name}}</a></li>
-   {{/if}}
+   {{#each groups}}
+      <li><a href="{{instanceURL}}/admin/permissions/data/group/{{this.id}}" style="color: #336699;">{{this.name}}</a></li>
+   {{/each}}
```

Comment:
I know it's churn, but TBH this is way more understandable to me.

Reply:
yea, I didn't even know `if` in mustache is also an iterator :). that's some dark magic.